app/src/main/res/layout/fragment_browser_tab.xml

```diff
@@ -38,7 +44,7 @@
         android:id="@+id/autoCompleteSuggestionsList"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="?attr/daxColorSurface"
+        android:background="?attr/daxColorBrowserOverlay"
```

Comment:
another color that uses background or surface based on the theme. creating a new overlay allows us to use the same layout for both themes.